internal/config/config.go

```diff
@@ -35,20 +33,15 @@ func NewConfig(rawConfig map[string]interface{}) ConfigFile {
 }
 
 func GetConfigPath() string {
-	configPath := os.Getenv("XDG_CONFIG_HOME")
-	if configPath == "" {
-		home, err := homedir.Dir()
-		if err != nil {
-			return ""
-		}
-		configPath = filepath.Join(home, ".config")
+	home, err := homedir.Dir()
```

Comment:
The main issue was that we would have to create the directory, so this simplifies things by removing that part and trying to save the file in the home directory. We could let people specify this at some point if we want.